frontend-ui/src/views/ScheduleDetailView.vue

```diff
@@ -832,6 +860,8 @@ onMounted(async () => {
       flagsDefinition.value = offlinerDefinition.flags
     }
     await fetchScheduleImageTags(schedule.value.config.image.name)
+    // Validate the schedule on mount
```

Comment:
Is this necessary since we already validate on `GET /schedule/xxx` in the backend?

Reply:
The rationale was that when a user fetches a schedule, we return the schedule if it exists. However, we only set the `is_valid` to notify them if the schedule meets current constraints. There is no way for them to know which fields are wrong with the schedule that was fetched. And I didn't want to return an error if a schedule was invalid.

So, on the UI, after fetching, we validate so we can see the specific fields that are wrong. They can then go to the Editor and resolve

Reply:
Done.